src/services/postLabels.ts

```diff
@@ -0,0 +1,40 @@
+import api from './api';
+import Router from 'next/router';
+import { toast } from 'react-toastify';
+
+import 'react-toastify/dist/ReactToastify.css';
+toast.configure()
+
+const postLabels = async (e, newLabel, oppenedBubbleId) => {
+  e.preventDefault();
```

Comment:
it would be better if we e.preventDefault() on onSubmit={e => {e.preventDefault(); callback()} so we dont have to pass it around